### DISCUSSION TITLE
2024/02/my-current-approach-to-software-delivery/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#874] [Specific Post Comments]
# 2024/02/my-current-approach-to-software-delivery/

As I prepare to onboard two new developers today (exciting!), part of the conversation will inevitably be around orienting them to my current philosophy of delivering software. I figured I’d share that here too.

https://seankilleen.com/2024/02/my-current-approach-to-software-delivery/

**Comment by dangros** [comment]
Well put. I’ve requested that we state the Prime Directive at the start of every retro so it helps set the tone for feedback and team safety.
